app/views/pledges/index.html.erb

```diff
@@ -1,5 +1,10 @@
 <div class="container">
-  <p id="notice"><%= notice %></p>
+  <% if notice %>
+    <div class="alert alert-success alert-dismissable">
```

Comment:
I did think about that, but it wasn't immediately apparent to me how the logic would work for that. I can give this another look.